Formula/ffmpeg@5.1.rb

```diff
@@ -0,0 +1,140 @@
+class FfmpegAT51 < Formula
```

Comment:
Seems nicer to call this `ffmpeg@5` for now, which will refer to the latest LTS 5.*.*, I think.

Reply:
But judging from the following, it seems only `5.1.*` are LTS? Looks unclear to me whether `5.2.*` will be LTS.

- https://git.ffmpeg.org/gitweb/ffmpeg.git/blob/refs/heads/release/5.1:/RELEASE_NOTES
- https://git.ffmpeg.org/gitweb/ffmpeg.git/blob/refs/heads/release/5.0:/RELEASE_NOTES
- https://endoflife.date/ffmpeg

Reply:
Sure, in which case `ffmpeg@5` is always 5.1.x.